src/client/connection.rs

```diff
@@ -42,3 +43,22 @@ where
         self.sender.send_request(req)
     }
 }
+
+impl<B> Service<Request<B>> for Connection<LiftBody<B>>
```

Comment:
I believe here we just want connection to be generic on `B` only where `B: Body`. We also then want `type Response = Response<B>`